src/ensembl/src/content/app/species/sample-data.ts

```diff
@@ -538,3 +540,179 @@ export const sampleData: RawSpeciesStats = {
     }
   }
 };
+
+type SpeciesSidebarData = {
+  [genomeId: string]: SpeciesSidebarPayload;
+};
+
+export const sidebarData: SpeciesSidebarData = {
+  homo_sapiens_GCA_000001405_28: {
+    id: 'GCA_000001405.28',
+    taxonomy_id: '9606',
+    strain: null,
+    database_version: '100.38',
+    common_name: 'Human',
+    scientific_name: 'Homo sapiens',
+    gencode_version: 'GENCODE 34',
+    assembly_name: 'GRCh38.p13',
+    assembly_provider: {
+      name: 'INSDC assembly',
+      url: ''
+    },
+    annotation_provider: {
+      name: 'Ensembl',
+      url: 'www.ensembl.org'
+    },
+    assembly_level: 'complete genome',
+    annotation_method: 'full genebuild',
+    assembly_date: '2019-02-01',
+    notes: [
+      {
+        heading: 'Pseudoautosomal regions',
+        body:
+          'The Y chromosome in this assembly contains two psuedoautosomal regions (PARs) that were taken from the corresponding X chromosoles and are exact duplicates:\nchrX:10,000-2,781,479 = chrY:10,000-2,781,479\nchrX:155,701,382-156,030,895 = chrY:56,887,02-57,217,415'
+      }
+    ]
+  },
+  homo_sapiens_GCA_000001405_14: {
+    id: 'GCA_000001405.14',
+    taxonomy_id: '9606',
+    strain: null,
+    database_version: '101.37',
+    common_name: 'Human',
+    scientific_name: 'Homo sapiens',
+    gencode_version: 'GENCODE 19',
+    assembly_name: 'GRCh37.p13',
+    assembly_provider: {
+      name: 'INSDC assembly',
+      url: ''
+    },
+    annotation_provider: {
+      name: 'Ensembl',
+      url: ''
+    },
+    assembly_level: 'complete genome',
+    annotation_method: 'full genebuild',
+    assembly_date: 'June 2013',
+    notes: []
+  },
+  escherichia_coli_str_k_12_substr_mg1655_GCA_000005845_2: {
+    id: 'GCA_000005845.2',
+    taxonomy_id: '511145',
+    strain: null,
+    database_version: '101.1',
+    common_name: 'Escherichia coli str. K-12 substr. MG1655',
```

Comment:
Umm, this is scientific name; common_name should be null.

Reply:
That's done

Reply:
Right. There's immediately a problem in the interface 😉 

![image](https://user-images.githubusercontent.com/6834224/97206071-2bb69100-17b0-11eb-8121-32825f79badd.png)

Reply:
Ah yes! Please wait until tomorrow before checking the UI :D

Reply:
That should now be fixed